ATL/schema.sql

```diff
@@ -0,0 +1,69 @@
+Create Table Passengers(
+Id int identity(1,1) Primary key,
+[Name] varchar(30),
+[Surname] varchar(50),
+Age int Not null,

```

Comment:
Storing age like this is a really bad idea. Try to think about why that is, we'll discuss it during the class.